### ISSUE TITLE
Execution failed for task ':sqlite3_flutter_libs:compileDebugJavaWithJavac'.

### ISSUE THREAD

**Issue by felipecastrosales** [simolus3/sqlite-native-libraries#3]
I have this error when build my app: 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sqlite3_flutter_libs:compileDebugJavaWithJavac'.
> Could not resolve all files for configuration ':sqlite3_flutter_libs:debugCompileClasspath'.
   > Failed to transform sqlite3-native-library-3.39.4.jar (eu.simonbinder:sqlite3-native-library:3.39.4) to match attributes {artifactType=android-classes-jar, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-api}.
      > Could not find sqlite3-native-library-3.39.4.jar (eu.simonbinder:sqlite3-native-library:3.39.4).
        Searched in the following locations:
            https://repo.maven.apache.org/maven2/eu/simonbinder/sqlite3-native-library/3.39.4/sqlite3-native-library-3.39.4.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

News about this?

**Comment by felipecastrosales** [comment]
@simolus3, What would be the best short-term solution to this?

**Comment by simolus3** [comment]
Apologies for this! The short-term solution is that I have retracted the broken package version on pub.dev. So if you run `pub get` again, you should get a working build again. I will update a fix soon (`3.39.4+1` should be synced to maven central soon :crossed_fingers:). I'll post more updates in https://github.com/simolus3/sqlite3.dart/issues/131, please follow that issue.
